gruntfile.js

```diff
@@ -83,21 +67,38 @@ module.exports = grunt => {
           },
         ],
       },
+      views: {
+        files: [
+          {
+            expand: true,
+            cwd: 'server/views',
+            src: ['./**/*'],
+            dest: 'dist/server/views/',
+          },
+        ],
+      },
```

Comment:
This replaces the `nodemon` `watch-view` script.

Instead of copying the view and restarting the server to load changes, this task watches and copies view changes and nunjucks will watch and automatically load view changes during live development. This makes view updates almost instant for local development.